flow.py

```diff
@@ -0,0 +1,122 @@
+class Flow(object):
+    """ A flow class that represents active connections between
+    hosts and routers.
+
+    Args:
+        ns: Instance of the NetworkSimulator class
+        flow_id: Unique id identifying the flow
+
+    Attributes:
+        flow_id: Unique id identifying the flow
```

Comment:
Put type after each attr, like flow_id (int): ...
Also no need to capitalize first word